### DISCUSSION TITLE
Compile error hidden by Cooja Simulation error

### DISCUSSION THREAD

**Discussion by rahav** [contiki-ng/contiki-ng#2670] [Q&A]
I receive the following display when loading a Cooja simulation:
![image](https://github.com/contiki-ng/contiki-ng/assets/2912950/0a381e5e-9da0-4bba-81ab-6317585c2e08)

At this point it is not clear what to do. The real problem will be shown only if the window above is closed:
![image](https://github.com/contiki-ng/contiki-ng/assets/2912950/079742cd-96e7-4d6b-b3a0-4f10f2f0bd37)

**Comment by pjonsson** [comment]
Isn't that fixed in 4.9? If it's not, do you still get the same issue on develop?
